shared/css/color.scss

```diff
@@ -135,7 +135,7 @@ $droplet_white: $white;
 
 // Oceans: deep blue background.
 $oceans_deep_blue: rgb(2, 0, 28);
-$music_black: #000;
+$music_black: #121212;
```

Comment:
I would also add a comment in the code (just like it's now for the` $oceans_deep_blue`: `// Oceans: deep blue background.)`)
That will say something like `// Music lab/dark theme black background`